src/app/pages/my-request-detail/my-request-detail.component.ts

```diff
@@ -122,10 +133,39 @@ export class MyRequestDetailComponent implements OnInit {
       )
       .subscribe((request) => {
         this.setRequest(request);
+        this.initialiseRequestQueries();
       });
     finalize(() => (this.state = 'idle'));
   }
 
+  private initialiseRequestQueries() {
+    if (!this.request?.id) {
+      return;
+    }
+
+    this.dataRequests
+      .getQuery(this.request.id, this.cohortQueryType)
+      .pipe()
```

Comment:
If there are no other RxJs operators to pipe, I think this empty `pipe()` call should be removed.

Reply:
fixed